### DISCUSSION TITLE
Passkey

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#719] [Q&A (Help)] [answered]
Hi, is there a plan to support the new Auth system Passkey ?
refs:
https://fidoalliance.org/passkeys/
https://developer.apple.com/passkeys/
https://developers.google.com/identity/passkeys
https://www.passkeys.io/

**Comment by tabacitu** [comment, marked as answer]
Hi @blondie63 ,

Wow, I didn't know passkey support advanced so far. YES, we do plan to support Passkeys. The soonest it can come is Feb 2024, in the next version, though.

I'll turn this into an issue, so I can put it on the roadmap. 
Thanks for opening it.

Cheers!

**Reply by tabacitu** [reply]
See https://github.com/Laravel-Backpack/community-forum/issues/721
